README.md

```diff
@@ -65,3 +65,73 @@ podman-compose -f deploy/podman/observability.yaml up
+
+COPY packaging/flightctl-custom-assets/flightctl_rsa.pub /usr/etc-system/root.keys
+RUN touch /etc/ssh/sshd_config.d/30-auth-system.conf; \
+    mkdir -p /usr/etc-system/; \
+    echo 'AuthorizedKeysFile /usr/etc-system/%u.keys' >> /etc/ssh/sshd_config.d/30-auth-system.conf; \
+    chmod 0600 /usr/etc-system/root.keys
+VOLUME /var/roothome
+
+ADD packaging/flightctl-custom-assets/config.yaml /etc/flightctl/
+ADD packaging/flightctl-custom-assets/ca.crt /etc/flightctl
+ADD packaging/flightctl-custom-assets/client-enrollment.* /etc/flightctl/
+
+RUN rpm-ostree install -y /tmp/flightctl-agent-0.0.1-1.el9.x86_64.rpm
+RUN ln -s /usr/lib/systemd/system/podman.socket /usr/lib/systemd/system/multi-user.target.wants/
+RUN ln -s /usr/lib/systemd/system/flightctl-agent.service /usr/lib/systemd/system/multi-user.target.wants/
+RUN ostree container commit 
+
+```
+
+In order to make this work for your own service, first you need to have deployed the service (by following the section above where the podman-compose manifests is applied).
+
+Execute `make rpm` to build the agent RPM package. It will be picked up and injected into the image.
+Now, add the following assets to the directory `packaging/flightctl-custom-assets/`:
+
+- The public SSH key you want to inject as `packaging/flightctl-custom-assets/flightctl_rsa.pub`
+- The CA certificate:  `packaging/flightctl-custom-assets/ca.crt`
+- The client enrollment keypair: 
+  - `packaging/flightctl-custom-assets/client-enrollment.crt`
+  - `packaging/flightctl-custom-assets/client-enrollment.key`
+- The configuration file that points to your service: `packaging/flightctl-custom-assets/config.yaml`
+
+It is important to highlight, that in a production environment, SSH will be disabled by default for security reasons. 
+The certificates will be stored probably in `$HOME/.flightctl/certs/`.
+
+The config.yaml should look like this:
+
+```
+agent:
+  server: https://localhost:3333
+  enrollementUi: https://localhost:3333
+  statusUpdateInterval: 1m0s
+  fetchSpecInterval: 1m0s
+```
+
+Change localhost by the IP of the machine where the service is exposed. Take into account that these services must be exposed to where the agent is going to be deployed.
+
+Now it's time to build the image! If you want to experiment with it, and push it to your own registry, you can do something like:
+
+```
+podman build -t quay.io/$MYQUAYUSER/flightctl-agent:latest -f packaging/Containerfile.fedora ./
+podman push quay.io/$MYQUAYUSER/flightctl-agent:latest
+```
+
+In order to make the deployment of this image easy, you can convert it to QCOW2 format and use it as a raw disk in Libvirt. The following command will create a file called disk.qcow2 within the output folder:
+
+```
+mkdir output && sudo podman run     --rm     -it     --privileged     --pull=newer     --security-opt label=type:unconfined_t     -v $(pwd)/output:/output     quay.io/centos-bootc/bootc-image-builder:latest   quay.io/$MYQUAYUSER/flightctl-agent:latest
+```
```

Comment:
```suggestion
mkdir output && sudo podman run --rm -it --privileged --pull=newer \
                                                    --security-opt label=type:unconfined_t \
                                                    -v $(pwd)/output:/output \
                                                    quay.io/centos-bootc/bootc-image-builder:latest \
                                                    quay.io/$MYQUAYUSER/flightctl-agent:latest
```

since I am in nit picking mode, feel free to ignore ':D

Reply:
welll it ended weird.. mehh

Reply:
Thanks, actually it had the indentation from the terminal separated in different lines, but I copied the wrong output!